.mergify.yml

```diff
@@ -2,26 +2,18 @@ pull_request_rules:
   - name: automerge to master with label automerge and branch protection passing
     conditions:
       - "#approved-reviews-by>1"
-      - base=master
+      - base=v0.45.0x-osmo-v7
```

Comment:
Can we just have `osmos-main` instead? What happens when we bump past v8? Seems error prone

Reply:
Agreed, we should make an `osmo-main`

Reply:
How would you like to proceed? Do you want to merge this first and then create the branch or create the the branch later and update the config again?

Reply:
Lets merge this first, then create the branch and update it again